editor-tool-highlighting/editor-tool-highlighting-ui-code/src/main/resources/SyntaxHighlighting/SyntaxHighlighting.xml

```diff
@@ -395,131 +379,6 @@ define(['jquery'], function ($) {
     }
   }
 
-  /* Synchronize the original textareas' contents with the codemirrors' contents just before saving the page. */
```

Comment:
Note that this change also means that 3rd party code using the "-code" module individually will now have to handle the save and fullscreen events on their own.